ripe/atlas/cousteau/source.py

```diff
@@ -127,6 +165,22 @@ def set_type(self, value):
     doc_type = "Defines the type of probe's source."
     type = property(get_type, set_type, doc=doc_type)
 
+    # tags attribute
+    def get_tags(self):
+        """Getter for tags attribute"""
+        return self._tags
+
+    def set_tags(self, value):
+        """Setter for tags attribute"""
+        super(AtlasChangeSource, self).set_tags(value)
+        if self.tags:
+            log = "Tag-based filtering can't be used when changing " \
+                  "participants probes for a measurement."
+            raise MalFormattedSource(log)
```

Comment:
You were right, it's not used. It will ignored most likely. In that case you don't need the call to the super. Just raise always the exception when they try to set and you are good to go.